### PR TITLE
Fixes for CentOS 8.2, CentOS7.8, Fedora31 images build. os-images role version bump, DIB version pinned

### PR DESCRIPTION
**Pull request by brtkwr**

### CODE REVIEW

File: etc/p3-config/p3-config.yml [hunk 1]
```diff
@@ -787,6 +819,7 @@ p3_image_mlnx_ofed:
     - "mlnx-ofed"
     - "selinux-permissive"
     - "alaska-extras"
+    - "openssh-server"
```
Comment (bbezak) [inline etc/p3-config/p3-config.yml]: is it really needed in centos7 cloud image?

File: etc/p3-config/p3-config.yml [hunk 2]
```diff
@@ -649,20 +650,51 @@ p3_image_fedora_coreos:
...
 
-# Latest CentOS image
+# Latest CentOS 8
 p3_image_centos:
+  name: "CentOS8.2"
+  elements:
+    - "centos"
+    - "epel"
+    - "selinux-permissive"
+    - "alaska-extras"
```
Comment (brtkwr) [inline etc/p3-config/p3-config.yml:670]: I had issues building the image using ``alaska-extras`` DIB element in its current form. Did you have success there?

File: etc/p3-config/p3-config.yml [hunk 3]
```diff
@@ -649,20 +650,51 @@ p3_image_fedora_coreos:
...
+    - "centos"
     - "epel"
     - "selinux-permissive"
     - "alaska-extras"
     - "dhcp-all-interfaces"
     - "vm"
+    - "dracut-regenerate"
+    - "cloud-init-datasources"
   env:
+    BASE_IMAGE_FILE: "CentOS-7-x86_64-GenericCloud-2003.qcow2.xz"
```
Comment (brtkwr) [inline etc/p3-config/p3-config.yml:704]: Would you also mind adding this base image to other Centos7 images?

File: etc/p3-config/p3-config.yml (hunk 1 above)
Comment (brtkwr) [inline etc/p3-config/p3-config.yml]: Do we still explicitly need ``openssh-server`` now that we are using ``centos`` element?

File: etc/p3-config/p3-config.yml [hunk 4]
```diff
@@ -649,20 +650,51 @@ p3_image_fedora_coreos:
...
 p3_image_centos:
+  name: "CentOS8.2"
+  elements:
+    - "centos"
+    - "epel"
+    - "selinux-permissive"
+    - "alaska-extras"
+    - "dhcp-all-interfaces"
+    - "vm"
+    - "dracut-regenerate"
```
Comment (brtkwr) [inline etc/p3-config/p3-config.yml:673]: Do we need dracut-regenerate in the cloud image?

Comment (bbezak) [reply etc/p3-config/p3-config.yml:673]: w need that in 8.2

**Review by bbezak — APPROVED** [review]
I think it looks good now

**Pull request merged**